deployment/changeset.go

```diff
@@ -113,8 +113,10 @@ type ChangesetOutput struct {
 	MCMSTimelockProposals      []mcms.TimelockProposal
 	DescribedTimelockProposals []string
 	MCMSProposals              []mcms.Proposal
-	AddressBook                AddressBook
-	DataStore                  datastore.MutableDataStore[datastore.DefaultMetadata, datastore.DefaultMetadata]
+	// Deprecated: AddressBook is deprecated and will be removed in future versions.
+	// Prefer the new DataStore API instead
```

Comment:
just nitpick: more like a convention to use `Use` 
```suggestion
	// Use DataStore instead
```